homeassistant/components/zwave_js/helpers.py

```diff
@@ -568,3 +574,23 @@ def get_network_identifier_for_notification(
             return f"`{config_entry.title}`, with the home ID `{home_id}`,"
         return f"with the home ID `{home_id}`"
     return ""
+
+
+async def async_get_version_info(hass: HomeAssistant, ws_address: str) -> VersionInfo:
```

Comment:
I moved this function and exception to helpers since it's now used both by the config flow and by the api module.